odoo/addons/test_translation_import/tests/test_term_count.py

```diff
@@ -341,3 +342,20 @@ def test_export_import_csv(self):
         })
         with mute_logger('odoo.addons.base.models.res_lang'):
             import_fr.with_context(create_empty_translation=True).import_lang()
+
+
+@tagged('post_install', '-at_install')
+class PotLinter(common.TransactionCase):
+    def test_pot_files(self):
+        for module in odoo.modules.get_modules():
+            filename = odoo.modules.get_resource_path(module, 'i18n', module + '.pot')
+            if not filename:
+                continue
+            counts = defaultdict(int)
+            for entry in TranslationFileReader(filename):
+                if entry['type'] == 'model':
+                    counts[(entry['name'], entry['imd_name'])] += 1
+                elif entry['type'] == 'model_terms':
+                    counts[(entry['name'], entry['imd_name'], entry['src'])] += 1
+            duplicates = [key for key, count in counts.items() if count > 1]
+            self.assertFalse(duplicates, "Duplicate entries found in %s" % filename)
```

Comment:
```suggestion
            counts = Counter(
                 (entry['name'], entry['imd_name']) if entry['type'] == 'model' else (entry['name'], entry['imd_name'], entry['src'])
                 for entry in TranslationFileReader(filename):
                 if entry['type'] in ('model', 'model_terms')
            )
            duplicates = any(count > 1 for _, count in counts.most_common(1))
            self.assertFalse(duplicates, "Duplicate entries found in %s" % filename)
```

Reply:
(could also use the same filter as the original if we want to show the duplicated entries)

Reply:
While the Counter is more pythonic, it lacks the list of duplicates keys

Reply:
As noted, you can just filter it as previously instead of using `most_common` and you'll get the same result.